### PR TITLE
Adds Item View, Fixes Date Format

### LINKED ISSUE #36: Listview - Irish Date Format
The current date format is mm/dd/yyyy - aka the American date format, please change it to dd/mm/yyyy

### PR DESCRIPTION
**Pull request by gkemp94**
Adds Item View on Click from List view and map view, also fixes date time formatting. 

fixes #36 and fixes #36

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
         "space-before-function-paren": "off",
         "prefer-destructuring": "off",
         "object-shorthand": "off",
+        "react/prop-types": ["error", { "ignore": ["navigation"] }],
```
Comment (gkemp94) [inline .eslintrc.json]: FYI: Ignoring navigation props in prop-types is standard procedure, navigation props will be supplied to any view in our navigation stack, and we shouldn't be concerned with their structure, just their use.

File: components/TwItemView.js [hunk 2]
```diff
@@ -40,7 +41,7 @@ class TwItemView extends React.Component {
         <Text note style={styles.title}>Description</Text>
         <Text>{alertItem.DESCRIPTION.replace(/<\/?[^>]+(>|$)/g, '')} </Text>
         <Text note style={styles.title}>Date</Text>
-        <Text>{new Date(alertItem.STARTDATE).toLocaleDateString()}</Text>
+        <Text>{formatDate(new Date(alertItem.STARTDATE))}</Text>
```
Comment (lukekearney) [inline components/TwItemView.js]: Do we need a custom function for this? Would `new Date(alertItem.STARTDATE).toLocaleDateString('en-GB')` work?

**Comment by lukekearney** [comment]
Please add a string to close any relevant issues in this pull request when this merges. Also, 

e.g. fixes #36

File: components/TwItemView.js (hunk 2 above)
Comment (gkemp94) [reply components/TwItemView.js]: That could work, but I don't believe it adds the extra 0's which Kat desired. So the custom function works to ensure that the date is formatted in that exact configuration.

Comment (SmilingKat) [reply components/TwItemView.js]: Hi @gkemp94 , 
can you check what the call @lukekearney proposed would display? 
Just testing it locally and attaching a photo here is sufficient for me.

Comment (gkemp94) [reply components/TwItemView.js]: @SmilingKat committing change now.

**Pull request merged**